### DISCUSSION TITLE
Arabic encoding

### DISCUSSION THREAD

**Discussion by RamiGamalMahmoud** [jchristn/SuperSimpleTcp#213] [Q&A]
**When I send text in Arabic to the printer, it appears like this.**
_This is my code_
`System.Text.Encoding.GetEncoding("UTF-8").GetBytes(arabicString),`

![IMG_20240808_134607](https://github.com/user-attachments/assets/1b99efc7-87ab-4aa9-87e9-af443b79b720)

**Comment by jchristn** [comment]
What if you try `Encoding.UTF8.GetBytes(arabicString)`?  Also there's not a lot of context here to understand what you're doing or why it may be happening.  Moving this to a discussion.
